src/core/control/Control.cpp

```diff
@@ -2373,3 +2378,22 @@ auto Control::getPageBackgroundChangeController() const -> PageBackgroundChangeC
 auto Control::getLayerController() const -> LayerController* { return this->layerController; }
 
 auto Control::getPluginController() const -> PluginController* { return this->pluginController; }
+
+auto Control::getPalette() const -> const Palette& { return *(this->palette); }
```

Comment:
- [ ] Is this the correct way to implement this? I'm not to sure since C++ is not my day-to-day language.

The intention is to simply return the reference to the one "global" `Palette` instance that's managed by `Control`. Basically a singleton since `Palette` is also instantiated only once.